guides/common/modules/proc_exporting-templates.adoc

```diff
@@ -1,7 +1,8 @@
 [id="Exporting_Templates_{context}"]
 = Exporting templates
 
-Use this procedure to export templates to a git repository.
+You can export templates to an existing repository.
+In this procedure, a Git repository is used for demonstration purposes.
```

Comment:
Probably not necessary to mention but that is more stylistic than technical thing so drop or keep at your own discretion.

Reply:
Let's drop it.